zerver/webhooks/zapier/doc.html

```diff
@@ -40,3 +40,36 @@
 <p>You're done! Example message:</p>
 
 <img class="screenshot" src="/static/images/integrations/zapier/005.png"/>
+
+<p>
+    <b>Weather Update with Zapier!</b><br />
+</p>
+<p>
+    Go to the following link <code>https://zapier.com/app/explore</code> after you sign into your Zapier account<br />
+   Select "Send weather info to your app via webhooks".
+
+    <img class="screenshot" src="/static/images/integrations/zapier/006.png" /><br />
+    Choose the trigger app <b>Weather by Zapier</b><br /><br />
+    <img class="screenshot" src="/static/images/integrations/zapier/007.png" /><br />
+    Next, select the option <b>Today's Forecast</b><br /><br />
```

Comment:
In the "Choose" sentence and the "Next" sentence, you should end each sentence with a period.